src/main/java/org/jabref/logic/util/io/FileUtil.java

```diff
@@ -329,9 +330,12 @@ public static String createFileNameFromPattern(BibDatabase database, BibEntry en
         if (targetName.isEmpty()) {
             targetName = entry.getCitationKey().orElse("default");
         }
-
+        // Remove LaTeX commands (e.g., \mkbibquote{}) from expanded fields before cleaning filename
+        // See: https://github.com/JabRef/jabref/issues/12188
+        targetName = new RemoveLatexCommandsFormatter().format(targetName);
```

Comment:
Please make this a class constant (`private static final ...`). This saves some time when executing this (we use this pattern in other places in JabRef)

Keep the empty line before.